test/run_envoy_bazel_coverage.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e -o pipefail
 
-LLVM_VERSION="14.0.0"
+LLVM_VERSION=${LLVM_VERSION:-"14.0.0"}
```

Comment:
@zirain no objections to this if it serves a need - but flagging that we are more trying to move towards using toolchains and away from host binaries

there are many reasons for this but not least the issue that we have the version set in a load of different places

Reply:
yeah i see running coverage directly on the host